version.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2013-2017 The btcsuite developers
+// Copyright (c) 2015-2016 The Decred developers
+// Heavily inspired by https://github.com/btcsuite/btcd/blob/master/version.go
+// Copyright (C) 2015-2020 The Lightning Network Developers
+
+package governator
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+// Commit stores the current commit hash of this build, this should be set using
+// the -ldflags during compilation.
+var Commit string
+
+// semanticAlphabet
```

Comment:
nit: incomplete comment? Can probably just delete it, pretty self explanatory var.

Reply:
Oops, copy paste. Fixed it.

Reply:
> copy paste

Friend or foe? We will never truly know.